ykrt/src/frame/mod.rs

```diff
@@ -175,6 +177,17 @@ impl FrameReconstructor {
             // reconstructed.
             if i > 0 {
                 memsize += usize::try_from(rec.size).unwrap();
+            } else {
+                // Get the size of the frame containing the control point, which we'll later use to
+                // calculate the stack offset to write the new frames to. Note that we'll be adding
+                // this to a pointer retrieved via the `llvm.frameaddr` intrinsic, which doesn't
+                // include the pushed RBP at the beginning of the frame, whereas the size reported
```

Comment:
RBP -> base pointer, since RBP is an x86_64 term and this file is (in theory) generic.

Reply:
The function is marked as `#[cfg(target_arch = "x86_64")]`, and I'm using the `RBP` term all over the place. I think it's fine.

Reply:
Ah, yes. OK!